controllers/managedosversionchannel_controller.go

```diff
@@ -140,7 +140,7 @@ func (r *ManagedOSVersionChannelReconciler) reconcile(ctx context.Context, manag
 		meta.SetStatusCondition(&managedOSVersionChannel.Status.Conditions, metav1.Condition{
 			Type:    elementalv1.ReadyCondition,
 			Reason:  elementalv1.InvalidConfigurationReason,
-			Status:  metav1.ConditionTrue,
+			Status:  metav1.ConditionFalse,
```

Comment:
This way it also show up in UI, this status true|fase was a bit inconsistent before. So it is only in ready state after a successful sync, and on creation syncing is the first thing it does. It don't think we need multiple conditions to track resource creation per se and synchronization status.

Also what UI tracks is the ready condition.